### ISSUE TITLE
Publisher element height and width in %

### ISSUE THREAD

**Issue by tushar-1health** [sean-perkins/nativescript-opentok#39]
Hello,

I stumbled upon a weird issue today. If we configure the element 
```
<TNSOTSubscriber #subscriber id="subscriber" width="1" height="1"></TNSOTSubscriber>
<TNSOTPublisher #publisher id="publisher" width="100%" height="100%"></TNSOTPublisher>
```
The app crashes in iOS device. However, as soon as we set a defined height and width to the publisher element it works out fine. 

Could you help me with it

**Comment by sean-perkins** [comment]
Does the crash produce any helpful error logs?

**Comment by tushar-1health** [comment]
It crashes w/ no errors in iOS (Device as well as Simulator). Android works fine with no errors and 100% heights and widths.

**Comment by triniwiz** [comment]
Fixed in #41
